deploy/table_emission.sql

```diff
@@ -8,13 +8,13 @@ create table ggircs.emission (
     id                        integer primary key,
     ghgr_import_id            integer,
     activity_id               integer references ggircs.activity(id),
-    single_facility_id        integer references ggircs.single_facility(id),
-    lfo_facility_id           integer references ggircs.lfo_facility(id),
+    facility_id               integer references ggircs.facility(id),
     fuel_id                   integer references ggircs.fuel(id),
     naics_id                  integer references ggircs.naics(id),
     organisation_id           integer references ggircs.organisation(id),
     report_id                 integer references ggircs.report(id),
     unit_id                   integer references ggircs.unit(id),
+    fuel_mapping_id           integer references ggircs_swrs.fuel_mapping,
```

Comment:
Why do we have a cross-schema reference here? Does fuel_mapping perhaps belong in `ggircs`?

Reply:
Also, `ggircs_swrs.fuel_mapping(id)` no?

Reply:
@wenzowski any mapping tables or tables that contain information like fuel charge I've left in the ggircs_swrs namespace as I felt it was information that doesn't need to be exposed, just used. So I figured it belonged in the private namespace. Does that make sense or should these kinds of table exist in the ggircs schema?

Reply:
Given that we might want to expose these via a front-end in the future, i think this table can live in ggircs

Reply:
should we leave it as is for now and make it part of a bigger overall namespace audit, (which I think we probably need sometime soon), or change it here in this PR?

Reply:
Great idea, ticket created https://youtrack.button.is/issue/GGIRCS-549

Reply:
> Also, `references ggircs_swrs.fuel_mapping(id)` no?